blackjack/blackjack.py

```diff
@@ -0,0 +1,217 @@
+from game import *
+from deck import *
+from card import *
+from player import *
+from dealer import *
+from hand import *
```

Comment:
I would avoid using `import *`. It brings everything into your current namespace and can result in confusing overriding of functions.